src/standard/expr.ml

```diff
@@ -1896,8 +1896,8 @@ module Term = struct
         [a; b] [Ty.of_var a] (Ty.of_var b)
 
     let in_interval (b1, b2) = mk'
-        ~name:"in_interval" ~builtin:(Builtin.In_interval (b1, b2))
-        "InInterval" [] [Ty.int; Ty.int; Ty.int] Ty.prop
+      ~name:"in_interval" ~builtin:(Builtin.In_interval (b1, b2))
+      "InInterval" [] [Ty.int; Ty.int; Ty.int] Ty.prop
```

Comment:
I believe this needs to be α → α → α, not `int → int → int` (at least it can also be `real → real → real`)

Reply:
Sure! I completely forgot it :)